Update SA1600 to show inheritance example. (#3385)
DotNetAnalyzers/StyleCopAnalyzers: documentation/SA1600.md

}
```

> The next example shows a method that is inherited from a base class:

Reviewer: ```suggestion
The next example shows a method that inherits documentation from a method in the base class:
```

Author: Looks good.

```csharp
/// <summary>
/// Example base class.
/// </summary>
class Vehicle 
{
  /// <summary>
  /// Accelerates the vehicle.
  /// </summary>

>   public virtual void accelerate()

Reviewer: ```suggestion
  public virtual void Accelerate()
```

Author: Looks good. I could not hide my java background :)

  {                    
    Console.WriteLine("Vehicle is accelerating");
  }
}

/// <summary>
/// Example derived class.
/// </summary>
class Car : Vehicle
{
  /// <inheritdoc/>

>   public override void accelerate()

Reviewer: ```suggestion
  public override void Accelerate()
```

Author: Looks good.
